libclamunrar_iface/CMakeLists.txt

```diff
@@ -109,6 +109,13 @@ if(ENABLE_UNRAR)
             PRIVATE
                 ClamAV::libunrar_static )
 
+        if(WIN32)
+            install(TARGETS clamunrar_iface_static DESTINATION . COMPONENT libraries)
+            install( FILES $<TARGET_PDB_FILE:clamunrar_iface> DESTINATION . OPTIONAL COMPONENT libraries )
```

Comment:
```suggestion
```
This line is not required. The target name is wrong, but also PDB files do not exist for static libraries.